### PR TITLE
SONAR-6353 do not compute the metric days_since_last_commit

### PR DESCRIPTION
**Pull request by simonbrandhof**
The metric days_since_last_commit was introduced during the 
sprint 5.2 but then was replaced by last_commit_date

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/computation/source/LastCommitVisitor.java [hunk 1]
```diff
@@ -41,7 +41,6 @@
   private final BatchReportReader reportReader;
   private final MeasureRepository measureRepository;
   private final Metric lastCommitDateMetric;
-  private final Metric daysSinceLastCommitDateMetric;
   private final System2 system2;
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/computation/source/LastCommitVisitor.java]: MAJOR Remove this unused "system2" private field. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Remove this unused "MILLISECONDS_PER_DAY" private field.](https://github.com/SonarSource/sonarqube/blob/d3309bab0750e72ae5282809c8d5efb223a75dbd/server/sonar-server/src/main/java/org/sonar/server/computation/source/LastCommitVisitor.java#L39) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)

**Comment by julienlancelot** [comment]
Sounds good, but... a (Java) migration is missing to purge measures. Indeed, as it was previously computed by Dev Cockpit, a lot of measures exist, so then should be purged.

**Comment by simonbrandhof** [comment]
Good catch. As discussed the measures won't be purged as they don't relate to core.

**Pull request closed without merging**